components/collective-page/BudgetStats.js

```diff
@@ -82,23 +82,28 @@ const BudgetStats = ({ collective, stats }) => {
       <StatContainer>
         <StatTitle>
           <ShowChart size="12px" />
-
-          <DefinedTerm
-            term={Terms.TOTAL_RAISED}
-            textTransform="uppercase"
-            color="black.700"
-            extraTooltipContent={
-              <Box mt={2}>
-                <FormattedMessage
-                  id="budgetSection-raised-total"
-                  defaultMessage="Total contributed before fees: {amount}"
-                  values={{
-                    amount: formatCurrency(stats?.totalAmountRaised.valueInCents || 0, collective.currency, { locale }),
-                  }}
-                />
-              </Box>
-            }
-          />
+          {collective.id === collective.host.id ? (
+            <DefinedTerm term={Terms.TOTAL_INCOME} textTransform="uppercase" color="black.700" />
```

Comment:
Can you add a tooltip explaining what this is?
Alanna suggested something like: `Net all time, including host fees and direct contributions.`

Reply:
Yes, this is already added within `DefinedTerm` so that takes care of it; 

https://github.com/opencollective/opencollective-frontend/blob/dd00fde8502345a8e580e85b88f4437d74549cc7/components/DefinedTerm.js#L96-L99